pkg/audit/manager.go

```diff
@@ -233,58 +255,70 @@ func (am *Manager) auditResources(ctx context.Context) ([]*constraintTypes.Resul
 		}
 	}
 
-	var responses []*constraintTypes.Result
 	var errs opa.Errors
 	nsCache := newNSCache()
 
 	for gv, gvKinds := range clusterAPIResources {
 		for kind := range gvKinds {
 			objList := &unstructured.UnstructuredList{}
-			objList.SetGroupVersionKind(schema.GroupVersionKind{
-				Group:   gv.Group,
-				Version: gv.Version,
-				Kind:    kind + "List",
-			})
-
-			err := am.client.List(ctx, objList)
-			if err != nil {
-				am.log.Error(err, "Unable to list objects for gvk", "group", gv.Group, "version", gv.Version, "kind", kind)
-				continue
+			opts := &client.ListOptions{
+				Limit: int64(*auditListLimit),
```

Comment:
what happens if limit is 0?

Reply:
setting to 0 disables chunking, documented here
https://github.com/open-policy-agent/gatekeeper/pull/734/files#diff-9dc803b1d6fa5cb358cdda0f674e4a34R46